scriptedforms/src/app/variables-module/variable-table.component.ts

```diff
@@ -41,10 +39,14 @@ import { PandasTable } from '../interfaces/pandas-table';
 
 import { VariableParameterComponent } from './variable-parameter.component';
 
+import * as htmlTemplate from 'html-loader!./variable-table.component.html';
+const template = '' + htmlTemplate;
```

Comment:
What's this for?

Reply:
For some reason the angular linter wasn't picking up the html template as a string. ... So... It was my quick attempt to make the linter be quiet. I'll have to work out a nicer way.

Reply:
It's actually got nothing to do with string or not. It's the Angular language service not recognising the template import. (Understandably as it is quite nonstandard).